src/views/catalog/CreateFromInstanceTypes/hooks/useBootableVolumes.ts

```diff
@@ -17,15 +26,29 @@ const useBootableVolumes: UseBootableVolumes = () => {
   >({
     groupVersionKind: DataSourceModelGroupVersionKind,
     isList: true,
+    namespace: isUpstream ? KUBEVIRT_OS_IMAGES_NS : OPENSHIFT_OS_IMAGES_NS,
     selector: {
       matchExpressions: [{ key: DEFAULT_PREFERENCE_LABEL, operator: Operator.Exists }],
     },
   });
 
+  const [pvcSources, setPVCSources] = useState<V1alpha1PersistentVolumeClaim[]>([]);
+
+  useEffect(() => {
+    if (loadedDataSources && !loadErrorDataSources && !isEmpty(dataSources)) {
+      const bla = dataSources.map((ds) =>
```

Comment:
bla bla bla lol

Reply:
Aviv, I know very well how it feels to always come up with some perfect and unique name, and to come up with that fast...
Maybe simply `pvcsArray` could be good to use as the name of the variable, for example? Or something like `pvcsFromDataSources` ? :) 🤔 You know, I'm trying to help :)

Reply:
lol changed